.vib/fluentd/goss/fluentd.yaml

```diff
@@ -0,0 +1,10 @@
+command:
+  check-no-pem-files:
+    exec: find /opt/bitnami/fluentd -name "*.pem" | grep .
+    exit-status: 1
+  check-fluent-plugin-detect-exceptions:
+    exec: ls -l /opt/bitnami/fluentd/gems/fluent-plugin-detect-exceptions-*/lib/fluent/plugin/*.rb 2>/dev/null | grep ".*-rwxrwxr-x"
+    exit-status: 0
+  check-plugins:
+    exec: ls /opt/bitnami/fluentd/gems/fluent-plugin-concat-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-detect-exceptions-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-elasticsearch-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-grafana-loki-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-kafka-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-kubernetes_metadata_filter-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-multi-format-parser-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-prometheus-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-rewrite-tag-filter-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-s3-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-systemd-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluent-plugin-record-modifier-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/oj-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/json-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/fluentd-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/activesupport-* 2>/dev/null && ls /opt/bitnami/fluentd/gems/elasticsearch-xpack-* 2>/dev/null
```

Comment:
This is valid but not an elegant approach IMHO.

1. Instead of a plain `ls`, you should check if the CLI does offer a similar functionality and prefer it. In this case, it seems that `fluent-gem` does so.
2. When you have a list or an iterable collection for which you want to perform the same check, you should use `go-template` loops to improve readability
3. Again, if you have a fixed list or collection related to test data, the right place to declare it is in `vars.yaml`. You can then reference it here, which is much cleaner 😁